composables/types/useCart.ts

```diff
@@ -0,0 +1,34 @@
+import { CrProduct } from "@/server/types/GraphQL"
+
+export type CrDiscount = {
+  id: String
+  name?: String
+}
+
+export type CrAppliedDiscount = {
+  couponCode?: [String]
+  discount?: CrDiscount
+  impact?: Number
+}
+
+export type CartItem = {
+  id: String
+  product: CrProduct
+  quantity: Number
+}
+
+export type Cart = {
```

Comment:
Don't recreate the Cart type here, just reuse the existing type(s) from graphql

Reply:
Not using this anymore. Deleted.